### PR TITLE
OSDOCS-9499# RWOP TP -> GA

### PR DESCRIPTION
**Pull request by lpettyjo**
Version(s): 4.16 

Issue: https://issues.redhat.com/browse/OSDOCS-9499

Link to docs preview: https://73710--ocpdocs-pr.netlify.app/openshift-enterprise/latest/storage/understanding-persistent-storage#pv-access-modes_understanding-persistent-storage

QE review:
- [x] QE has approved this change.

Additional information:

PTAL: @jsafrane, @chao007, @gcharot

### CODE REVIEW

File: modules/storage-persistent-storage-pv.adoc [hunk 1]
```diff
@@ -172,26 +168,24 @@ ifdef::openshift-enterprise,openshift-webscale,openshift-origin[]
...
 endif::[]
 |===
 [.small]
 --
 1. ReadWriteOnce (RWO) volumes cannot be mounted on multiple nodes. If a node fails, the system does not allow the attached RWO volume to be mounted on a new node because it is already assigned to the failed node. If you encounter a multi-attach error message as a result, force delete the pod on a shutdown or crashed node to avoid data loss in critical workloads, such as when dynamic persistent volumes are attached.
 
-2. ReadWriteOncePod is a Technology Preview feature.
-
-3. Use a recreate deployment strategy for pods that rely on AWS EBS.
+2. Use a recreate deployment strategy for pods that rely on AWS EBS.
```
Comment (mramendi) [inline modules/storage-persistent-storage-pv.adoc:183]: The style guide says re-create not recreate: https://www.ibm.com/docs/en/ibm-style?topic=word-usage (search for "re-create"). However, this is pre-existing text, so this is really out of scope; if it is possible that the UI or config uses "recreate", leave this be.

**Review by mramendi — APPROVED** [review]
Only one nit, and it might be out of scope. As I am in peer review induction, @JoeAldinger will finish this review

Also: I assume you did get QE approval for this? Ideally please mark in a comment who in QE approved the change. We do have to get QE approval for anything except pure editorial change.

**Review by JoeAldinger — COMMENTED** [review]
Small suggestion for CI item while you are here. Otherwise, lgtm

**Review by mletalie — COMMENTED** [review]
/remove-label peer-review-needed

**Comment by jsafrane** [comment]
lgtm

**Comment by jsafrane** [comment]
lgtm

**Comment by gcharot** [comment]
lgtm

File: modules/storage-persistent-storage-pv.adoc [hunk 2]
```diff
@@ -126,7 +126,7 @@ The following table lists the access modes:
 |ReadWriteOnce
 |`RWO`
 |The volume can be mounted as read-write by a single node.
-|ReadWriteOncePod ^[1]^
+|ReadWriteOncePod
```
Comment (gcharot) [inline modules/storage-persistent-storage-pv.adoc:129]: Do we want to add selinux context mount here as a note?

**Comment by gcharot** [comment]
lgtm as it is but do we want to add selinux context mount note as part of this doc PR?

File: modules/storage-persistent-storage-pv.adoc (hunk 2 above)
Comment (lpettyjo) [reply modules/storage-persistent-storage-pv.adoc]: Yes. Done.

**Comment by lpettyjo** [comment]
SELinux content added for https://issues.redhat.com/browse/STOR-1464. @jsafrane @radeore  @gcharot please review new content. Thanks!

File: modules/storage-persistent-storage-pv.adoc [hunk 3]
```diff
@@ -139,29 +139,27 @@ ifdef::openshift-enterprise,openshift-webscale,openshift-origin[]
 endif::[]
 |===
 --
-1. ReadWriteOncePod access mode for persistent volumes is a Technology Preview feature.
+1. RWOP uses the SELinux mount feature. Supported vendors include EBS, Azure Disk, GCP PD, IBM VPC Block, Cinder, and vSphere.
```
Comment (gcharot) [inline modules/storage-persistent-storage-pv.adoc:142]: RWOP uses the SELinux context mount feature.
Maybe add ODF as well.

Comment (chao007) [reply modules/storage-persistent-storage-pv.adoc:142]: Suggest to update as  "include AWS EBS xxx".

Comment (gcharot) [reply modules/storage-persistent-storage-pv.adoc:142]: confirmed with the ODF PM, they want us to add openshift data foundation to the list

Comment (lpettyjo) [reply modules/storage-persistent-storage-pv.adoc:142]: Done.

**Comment by lpettyjo** [comment]
@jsafrane, @chao007, @gcharot "EBS" changed to "AWS EBS" and ODF added. PTAL one last time!

**Comment by chao007** [comment]
lgtm

File: modules/storage-persistent-storage-pv.adoc [hunk 4]
```diff
@@ -136,26 +136,24 @@ ifdef::openshift-enterprise,openshift-webscale,openshift-origin[]
 endif::[]
 |===
 --
-1. ReadWriteOncePod access mode for persistent volumes is a Technology Preview feature.
+1. RWOP uses the SELinux mount feature. Supported vendors include ODF, AWS EBS, Azure Disk, GCP PD, IBM VPC Block, Cinder, and vSphere.
```
Comment (gcharot) [inline modules/storage-persistent-storage-pv.adoc:139]: RWOP uses the SELinux **context** mount feature
Also the vendor list only includes drivers we ship, we should add a sentense for third party drivers.

`This feature is driver dependent and enabled by default in ODF, AWS EBS, Azure Disk, GCP PD, IBM VPC Block, Cinder, and vSphere. For third party drivers please contact your storage vendor.`

Comment (lpettyjo) [reply modules/storage-persistent-storage-pv.adoc:139]: Done.

**Comment by jsafrane** [comment]
Lgtm
